docs/code_overview/other.rst

```diff
@@ -50,5 +51,6 @@ instances of them bound to an attribute of one of the PRAW models.
    other/domainlisting
    other/listinggenerator
    other/redditbase
+   other/redditorlist
```

Comment:
I've added RedditorList in this updated commit becuase RedditorList is refrerenced [here](https://github.com/praw-dev/praw/pull/705/commits/98863581eda64469854079f8836f04ceb5f56f2f#diff-ddfe088db1ec17b1133e057c48bd948cR101). Is it appropriate?